test/fixtures.py

```diff
@@ -4,27 +4,46 @@
 import logging
 import os
 import os.path
-import shutil
+import random
+import socket
+import string
 import subprocess
-import tempfile
 import time
 import uuid
 
-from six.moves import urllib
+import py
```

Comment:
What benefit do we get from this new dependency `py`?

Reply:
I had a look at replacing `tempdir` with the pytest's `tmpdir` fixture as you suggested and I liked the idea.

The pytest's `tmpdir` fixture is an instance of the `py.path.local` class. Once I finish converting all the test cases to `pytest`, the `zookeeper` and `kafka` fixtures will automatically get a `tmpdir` fixture from `pytest` and will pass it to the `ZookeeperFixture` and `KafkaFixture` constructors, respectively.

Before the conversion is complete, though, the constructors` will still not being passed `tmpdir` as a parameter and will have to instantiate it internally.

Hence, this import is only needed temporarily to allow the `unittest` code to continue working while we're going through this conversion process. Once everything is converted to `pytest`, we can remove this since the `tmpdir` will always be provided by the `pytest` fixtures.